### DISCUSSION TITLE
Ring Alarm modes versus Homekit Automations

### DISCUSSION THREAD

**Discussion by t829913** [dgreif/ring#1017] [Q&A]
With our existing Ring setup, we set to 'Away' when we go out, and 'Home' when we return. We almost never set it to 'Disarmed'.

When adding Homekit automations via the Home app, I can only choose from 'Triggers', 'Arms' or 'Disarms' as the states for the alarm. How do these options map to Away / Home / Disarmed? From the experimenting I've done it seems that both 'Home' and 'Away' both count as 'Arms' so I can't figure out how to build automations based on these states... help?

**Comment by dgreif** [comment]
The typical setup is to use Away when you are not home, Home when you are home but do not plan to leave (like when you are sleeping and the doors are locked), and Disarmed when you are home and moving about. With that in mind, I'm guessing you might just need to change your current "Home" setup to be "Disarmed".

You are correct that both Home and Away are "armed" triggers. If you are using these to trigger _other_ devices, then you unfortunately won't be able to do a more granular choice.

That said, the typical setup is not to use the Ring state as a _trigger_, but rather to _set_ it with automations. When setting it, you do get the choice of all three options (home/away/off).
